### DISCUSSION TITLE
Изменение поля сортировки при открытии таблицы через linkToTable

### DISCUSSION THREAD

**Discussion by an220022** [totumonline/totum-ru-issues-and-discussions#137] [Как это сделать?] [answered]
Как-то можно сделать чтобы при открытии простой таблицы через linkToTable поле и порядок сортировки строк таблицы менялась?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Через скрытое поле префильтра и код форматирования таблицы:

<img width="849" alt="Снимок экрана 2022-05-20 в 14 23 37" src="https://user-images.githubusercontent.com/55755565/169518135-683c1c51-1a1f-4b33-af5e-a1f6c17adcfb.png">


Код вызова окна:

```
a1=: linkToTable(table: 'stock'; title: "All in stock"; filter: 'fl_asc' = true; target: "iframe"; width: 1000; refresh: true)
```


Результат:

<img width="1033" alt="Снимок экрана 2022-05-20 в 14 24 38" src="https://user-images.githubusercontent.com/55755565/169518221-6e9c5d40-8662-4cd6-9542-f872b06dee51.png">
